arrr.py

```diff
@@ -102,6 +103,7 @@
     "blisterin' barnacles!",
     "ye flounderin' nincompoop.",
     "thundering typhoons!",
+    "sling yer hook"
```

Comment:
Missing comma at the end.

Not strictly needed, but its best practice to end a list with the last item having a comma.

Reply:
This should now be correct

Reply:
:+1: